tools/pd-simulator/simulator/config.go

```diff
@@ -41,20 +44,43 @@ const (
 	defaultLeaderPriorityCheckInterval = 100 * time.Millisecond
 )
 
+// unit of storage
```

Comment:
https://github.com/tikv/pd/blob/34a4cce2cb07f8aa4283e28ba40a667e1321a1fb/tools/pd-simulator/simulator/cases/cases.go#L61 it is repeated

Reply:
replace with unit.MiB .